<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/prefix1/icons/alarm-clock.png</normaloff>:/new/prefix1/icons/alarm-clock.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>521</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="actionRemove"/>
   <addaction name="separator"/>
   <addaction name="actionEdit"/>
   <addaction name="actionClone_2"/>
   <addaction name="separator"/>
   <addaction name="actionClose"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>590</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menu_Alarm">
    <property name="title">
     <string>&amp;Alarm</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionRemove"/>
    <addaction name="separator"/>
    <addaction name="actionEdit"/>
    <addaction name="actionClone_2"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionFloating_window_clock"/>
    <addaction name="actionBacground_options"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Alarm"/>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/icons/add_alarm.png</normaloff>:/new/prefix1/icons/add_alarm.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/icons/remove_alarm.png</normaloff>:/new/prefix1/icons/remove_alarm.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/icons/close_program.png</normaloff>:/new/prefix1/icons/close_program.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/icons/edit_alarm.png</normaloff>:/new/prefix1/icons/edit_alarm.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionClone_2">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/icons/clone_alarm.png</normaloff>:/new/prefix1/icons/clone_alarm.png</iconset>
   </property>
   <property name="text">
    <string>Clone</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionFloating_window_clock">
   <property name="text">
    <string>Floating window clock</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/icons/save_file.png</normaloff>:/new/prefix1/icons/save_file.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/icons/open_file.png</normaloff>:/new/prefix1/icons/open_file.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/icons/save_as.png</normaloff>:/new/prefix1/icons/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionBacground_options">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/icons/theme.png</normaloff>:/new/prefix1/icons/theme.png</iconset>
   </property>
   <property name="text">
    <string>Background options</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
